include/sicks3000.h

```diff
@@ -66,6 +66,9 @@ class SickS3000
     //! Get the current error cause and suggested action, if an error exists
     int GetDeviceError(uint8_t first_char, uint8_t second_char, std::string* error, std::string* suggested_action);
 
+    //! Initiate Read - required call prior to ReadLaser!
+    int InitiateRead(sensor_msgs::LaserScan& scan);
```

Comment:
Why is this new function necessary? I'm not seeing the case for why it is necessary.

(And if it is, why aren't you calling it from `s3000_laser.cc`?